plan/plans/show.go

```diff
@@ -528,12 +515,12 @@ func (s *ShowPlan) fetchShowStatus(ctx context.Context) error {
 
 		var value string
 		if !s.GlobalScope {
-			value, err = getSessionStatusVar(ctx, sessionVars, globalVars, v.Name)
+			value, err = getSessionStatusVar(ctx, sessionVars, status, v)
```

Comment:
But sessionVars values are different for each session. I think it can be changed when GetStatusVars has the ctx argument. And do it in next pr is better?